### PR TITLE
Dump start stop

### PR DESCRIPTION
**Pull request by magnusbechwind**
## Purpose

Implements dumpStart and dumpStop

## Changes

Implemented the above methods

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: concordium-sdk/src/main/java/com/concordium/sdk/ClientV2.java [hunk 1]
```diff
@@ -219,6 +221,31 @@ public TransactionStatus getBlockItemStatus(Hash transactionHash) throws BlockNo
         }
     }
 
+    /**
+     * Start dumping packages into the specified {@link File}
+     * Only enabled if the node was built with the 'network_dump' feature enabled
+     *
+     * @param dumpRequest {@link DumpRequest} specifying the file and if raw packages should be dumped
+     * @throws {@link io.grpc.StatusRuntimeException} if the network dump failed to start
```
Comment (EmilLa1) [inline concordium-sdk/src/main/java/com/concordium/sdk/ClientV2.java:229]: ... Or if the node was not build with the feature right?

File: concordium-sdk/src/main/java/com/concordium/sdk/requests/dumpstart/DumpRequest.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+public class DumpRequest {
+
+    /**
+     * Which file to dump the packages into
+     * Requires a valid path
+     */
+    private final File file;
+
+    /**
+     * Whether the node should dump raw packages
```
Comment (EmilLa1) [inline concordium-sdk/src/main/java/com/concordium/sdk/requests/dumpstart/DumpRequest.java:23]:
```suggestion
     * Whether the node should dump raw packages.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 ## Unreleased changes
+- Added support for GRPC V2 Dump Start
```
Comment (EmilLa1) [inline CHANGELOG.md:4]:
```suggestion
- Added support for GRPC V2 `DumpStart` which instructs the node to start dumping network packages at a specified path.
```

File: concordium-sdk/src/main/java/com/concordium/sdk/requests/dumpstart/DumpRequest.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+ */
+@Builder
+@Getter
+public class DumpRequest {
+
+    /**
+     * Which file to dump the packages into
+     * Requires a valid path
+     */
+    private final File file;
```
Comment (EmilLa1) [inline concordium-sdk/src/main/java/com/concordium/sdk/requests/dumpstart/DumpRequest.java:20]: I think a `Path` would be the better option here, as the SDK retains no handle to the file that is specified.

**Review by EmilLa1 — CHANGES_REQUESTED** [review]
Looks good!

A few nitpicky things and it's good to go.

**Review by abizjak — APPROVED** [review]

**Review by EmilLa1 — APPROVED** [review]

**Pull request merged**